website_docs/getting-started.md

```diff
@@ -3,552 +3,920 @@ title: "Getting Started"
+        "SpanKind": 1,
+        "StartTime": "0001-01-01T00:00:00Z",
+        "EndTime": "0001-01-01T00:00:00Z",
+        "Attributes": null,
+        "Events": null,
+        "Links": null,
+        "Status": {
+                "Code": "Unset",
+                "Description": ""
+        },
+        "DroppedAttributes": 0,
+        "DroppedEvents": 0,
+        "DroppedLinks": 0,
+        "ChildSpanCount": 0,
+        "Resource": [
+                {
+                        "Key": "environment",
+                        "Value": {
+                                "Type": "STRING",
+                                "Value": "getting-started"
+                        }
+                },
+                {
+                        "Key": "service.name",
+                        "Value": {
+                                "Type": "STRING",
+                                "Value": "diceroller-service"
+                        }
+                },
+                {
+                        "Key": "service.version",
+                        "Value": {
+                                "Type": "STRING",
+                                "Value": "v0.1.0"
+                        }
+                },
+                {
+                        "Key": "telemetry.sdk.language",
+                        "Value": {
+                                "Type": "STRING",
+                                "Value": "go"
+                        }
+                },
+                {
+                        "Key": "telemetry.sdk.name",
+                        "Value": {
+                                "Type": "STRING",
+                                "Value": "opentelemetry"
+                        }
+                },
+                {
+                        "Key": "telemetry.sdk.version",
+                        "Value": {
+                                "Type": "STRING",
+                                "Value": "1.7.0"
+                        }
+                }
+        ],
+        "InstrumentationLibrary": {
+                "Name": "diceroller-service",
+                "Version": "",
+                "SchemaURL": ""
+        }
+}
+```
 
-# Putting It All Together
+</details>
+
+You'll find that the `dice-roller` span has information about a parent span, and
+the ID of that parent span matches the span created by automatic
+instrumentation.
+
+## Send traces to an OpenTelemetry Collector
+
+The [OpenTelemetry Collector](/docs/collector/getting-started/) is a critical
+component of most production deployments. Some examples of when it's beneficial
+to use a collector:
+
+* A single telemetry sink shared by multiple services, to reduce overhead of
+  switching exporters
+* Aggregating traces across multiple services, running on multiple hosts
+* A central place to process traces prior to exporting them to a backend
+
+Unless you have just a single service or are experimenting, you'll want to use a
+collector in production deployments.
```

Comment:
Oh, this text is repeated across getting-started pages, right (I've suggested some edits to another page)! We might want to create a shortcode for the shared content. WDYT @cartermp?

Reply:
I think a subpage would be great, yeah. Or whatever thing works best. I don't know if that works across repos, though.